### ISSUE TITLE
coqide on Windows has trouble with _CoqProject local files

### ISSUE THREAD

**Issue by andrew-appel** [coq/coq#15670]
On Windows, coqide handles certain parts of `_CoqProject` all right (such as -Q options), but has trouble with names of local files.

For example, this _CoqProject file worked fine years ago (probably still works), and others like it work well now:

`-Q . VC  -Q C:/Local/vst2 VST -Q C:/Local/vst2/compcert compcert`

But in my Windows install of the Coq Platform 8.15 beta package pick, I have trouble with this _CoqProject:

```
a.v
b.v
```

When I start up `coqide a.v`, in the bottom of the window there is briefly a message about

`Load failed ... parsing error ... unsupported file name, contains "\\" ... C:\\\\Users\\\\Appel...`

(This message disappears after 4 seconds, so I can't reproduce it exactly.  That's also a bug, that coqide displays an error message for 4 seconds and then overwrites it, and there's no documented way to output a log of the error messages.  Or rather, there might be a documented way via `coqide --help`, but see issue #15586)

It appears that coqide is doing a `pwd` to get an absolute path name, then can't understand the path name given to it by Windows.

I imagine that this is not a new problem in Coq 8.15.   The reason I haven't noticed this, in the last ten years, is that I have used _CoqProject only for handling the -Q options, and never for actually listing my source files.

If I change the _CoqProject to the following, then it works fine.
```
/home/appel/test-coqproject/a.v
/home/appel/test-coqproject/b.v
```
But of course, that's not very portable for installing the _CoqProject in a git repo, which someone else may download in another directory.   

By the way, the cygwin `pwd` gives me `/home/appel/test-coqproject` instead of `C:\\path\to\test-coqproject`; and coqide works fine with that kind of pathname; so maybe that's a hint about how to resolve this situation.

**Comment by andrew-appel** [comment]
Why is it even necessary for CoqIDE to parse the details of the "files" part of the _CoqProject?   I would think that (perhaps) the only part CoqIDE needs is the imports part, i.e.,  -Q and -R etc.

**Comment by Alizter** [comment]
cc @jfehrle and @coq/coqide-maintainers 

Do you remember changing how the windows file separator was parsed in 8.15?

@andrew-appel I think that you are entirely correct that CoqIDE could be completely ignoring file names altogether since that is only used by coq_makefile.

**Comment by jfehrle** [comment]
I believe the issue appeared in 8.15.0 from https://github.com/coq/coq/pull/14558 and that it's fixed by https://github.com/coq/coq/pull/15775 in 8.15.1.

**Comment by jfehrle** [comment]
Note that pathnames in _CoqProject must only use '/', even on Windows.  I believe cygwin accepts paths such as `C:/home/jfehr` BTW.

**Comment by andrew-appel** [comment]
I confirm that this is, indeed, fixed in Coq 8.15.1.  Thank you.
